File: customized-checkout/payment/others/ticket/python/create_payment.py
import mercadopago
import json

mp = mercadopago.MP("ACCESS_TOKEN")

payment = mp.post("/v1/payments", {
        "transaction_amount": 100,
        "description": "Title of what you are paying for",
        "payment_method_id": "TICKET_PAYMENT_METHOD_ID",
        "payer": {
            "email": "test_user_19653727@testuser.com"
        }
    });

print(json.dumps(payment, indent=4))

File: customized-checkout/payment/basic/nodejs/create_payment.js
var MP = require ("mercadopago");

var mp = new MP ("ACCESS_TOKEN");

var doPayment = mp.post ("/v1/payments",
	{
		"transaction_amount": 100,
		"token": "ff8080814c11e237014c1ff593b57b4d",
		"description": "Title of what you are paying for",
		"installments": 1,
		"payment_method_id": "visa",
		"payer": {
			"email": "test_user_19653727@testuser.com"
		}
	});

doPayment.then (
	function (payment) {
		console.log (payment);
	},
	function (error){
		console.log (error);
	});